chia/cmds/wallet.py

```diff
@@ -75,8 +75,11 @@ def get_transactions_cmd(wallet_rpc_port: int, fingerprint: int, id: int, offset
     required=True,
 )
 @click.option("-t", "--address", help="Address to send the XCH", type=str, required=True)
-def send_cmd(wallet_rpc_port: int, fingerprint: int, id: int, amount: str, fee: str, address: str) -> None:
-    extra_params = {"id": id, "amount": amount, "fee": fee, "address": address}
+@click.option("-c", "--confirm", help="Submits transaction without checking for unusual values", is_flag=True)
```

Comment:
What is the default value here?

Reply:
defaults to False, I updated it to be explicit